src/App.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useState } from "react";
+import Edit from "./Image/edit.png";
+import monthName from "./Components/Monthname";
+import { ButtonEdit, Button } from "./StyleComponents/Button";
+import Clock from "react-live-clock";
+import Footer from "./StyleComponents/Footer";
+import { allUpdateNote } from "./Type/Type";
+import GlobalStyle from "./globalStyle";
```

Comment:
Please use pascal case for all filenames. Why you are not consistent on this, we discussed this in the review meeting ?